testsuite/MDAnalysisTests/core/test_topologyobjects.py

```diff
@@ -275,6 +279,52 @@ def test_bad_creation_TG(self):
         inputlist = ['a', 'b', 'c']
         assert_raises(TypeError, TopologyGroup, inputlist)
 
+    def test_tg_creation_bad_btype(self):
+        vals = np.array([[0, 10], [5, 15]])
+
+        assert_raises(ValueError, TopologyGroup, vals, self.universe, btype='apple')
+
+    def test_bond_tg_creation_notype(self):
+        vals = np.array([[0, 10], [5, 15]])
+
+        tg = TopologyGroup(vals, self.universe)
+
+        assert_(tg.btype == 'bond')
+        assert_array_equal(tg[0].indices, (0, 10))
+        assert_array_equal(tg[1].indices, (5, 15))
+
+    def test_angle_tg_creation_notype(self):
+        vals = np.array([[0, 5, 10], [5, 10, 15]])
+
+        tg = TopologyGroup(vals, self.universe)
+
+        assert_(tg.btype == 'angle')
+        assert_array_equal(tg[0].indices, (0, 5, 10))
+        assert_array_equal(tg[1].indices, (5, 10, 15))
+
+    def test_dihedral_tg_creation_notype(self):
+        vals = np.array([[0, 2, 4, 6], [5, 7, 9, 11]])
+
+        tg = TopologyGroup(vals, self.universe)
+
+        assert_(tg.btype == 'dihedral')
+        assert_array_equal(tg[0].indices, (0, 2, 4, 6))
+        assert_array_equal(tg[1].indices, (5, 7, 9, 11))
+
+    def test_create_guessed_tg(self):
+        vals = np.array([[0, 10], [5, 15]])
+
+        tg = TopologyGroup(vals, self.universe, guessed=True)
+
+        assert_array_equal(tg._guessed, np.array([[True], [True]]))
+
+    def test_create_guessed_tg_2(self):
+        vals = np.array([[0, 10], [5, 15]])
+
+        tg = TopologyGroup(vals, self.universe, guessed=False)
```

Comment:
What does the `guessed` keyword do for `TopologyObject`s? I don't see it documented in the class or `__init__` docs.